core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/exercise/ExerciseActionButtons.kt

```diff
@@ -187,4 +204,30 @@ class BoundExerciseActions(
         onClickViewResult = { onClickViewResult(exerciseId) },
         onClickViewQuizResults = { onClickViewQuizResults(exerciseId) }
     )
-}
\ No newline at end of file
+}
+
+
+@Composable
+fun InfoMessageCard() {
+    Box(
+        modifier = Modifier
+            .border(width = 2.dp, color = Color.LightGray)
+            .background(Color(0xFFB3E5FC)) // Light sky blue background
```

Comment:
The app supports both light an dark theme. Hardcoding color values may look bad on the dark theme.